go/healthserver/api.go

```diff
@@ -0,0 +1,43 @@
+/*
+	Package healthserver registers HTTP handlers for health and readiness checks
+
+	Package healthserver registers HTTP handlers for the /healthz and
+	/readiness paths. These handlers are always registered (this is done at
+	package init time).
+
+	By default, the /health handler responds with "OK".
+
+	By default, the /readiness handler responds with a HTTP header with
```

Comment:
HTTP headers are generally key value pairs separated by colon like Content-Type: text/html. Maybe http status instead of http header?